pkg/utils/ipset.go

```diff
@@ -395,20 +398,45 @@ func parseIPSetSave(ipset *IPSet, result string) map[string]*Set {
 	return sets
 }
 
+const tmpIPSetPrefix = "TMP-"
```

Comment:
nitpic: can we move this up to the const section at the top of the file?

Reply:
@aauren done.